src/test/scala/edu/ie3/simona/test/common/input/LoadInputTestData.scala

```diff
@@ -35,4 +39,23 @@ trait LoadInputTestData extends NodeInputTestData {
   )
 
   protected val loadInputContainer = SimpleInputContainer(loadInput)
+
+  private val operationTimeBuilder = OperationTime.builder()
+
+  private val interval = new ClosedInterval[ZonedDateTime](
+    TimeUtil.withDefaults.toZonedDateTime("2020-01-01T10:00:00Z"),
+    TimeUtil.withDefaults.toZonedDateTime("2020-01-01T13:00:00Z"),
+  )
+  private val operationTime: OperationTime =
+    operationTimeBuilder.withOperationTime(interval).build()
```

Comment:
Not to be overly nitpicky here, but this could be a bit more concise :)

```suggestion
  private val operationTime: OperationTime =
    OperationTime
      .builder()
      .withStart(TimeUtil.withDefaults.toZonedDateTime("2020-01-01T10:00:00Z"))
      .withEnd(TimeUtil.withDefaults.toZonedDateTime("2020-01-01T13:00:00Z"))
      .build()
```